### ISSUE TITLE
Troubles install SCS plugin on Ubuntu 18.04

### ISSUE THREAD

**Issue by plavjanik** [zowe/zowe-cli#1173]
I have following Linux system:
```
node -v
v16.12.0
zowe --version
6.36.0
npm --version
8.1.0
. /etc/os-release; echo $VERSION
18.04.6 LTS (Bionic Beaver)
```

I have installed Zowe using: `npm -g install @zowe/cli` and I am trying to install SCS plugin:

```sh
 zowe plugins install @zowe/secure-credential-store-for-zowe-cli@zowe-v1-lt
```

I am getting the following error:
```
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '/usr/lib/node_modules'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
Command Error:
Install Failed
```

The directory exists:
```
ls /usr/lib/node_modules
@zowe  corepack  npm
```

**Comment by t1m0thyj** [comment]
@plavjanik Thanks for reporting, this should be fixed in `@zowe/cli@6.36.1`. See the related issue https://github.com/zowe/imperative/issues/683

**Comment by plavjanik** [comment]
Thank you
